### PR TITLE
Merging Profile Branch into main

### PR DESCRIPTION
**Pull request by brvarner**

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      <div class="card-body">
+        <h5 class="card-title"><%= photo.owner.username %></h5>
+        <p class="card-text"><%= photo.caption %></p>
+      </div>
+      <ul class="list-group list-group-flush">
+        <% photo.comments.each { |comment| %>
+          <li class="list-group-item">
+            <div class="d-flex">
+              <div class="flex-shrink-0">
+                <img src="" alt="">
```
Comment (JoseMDO) [inline app/views/users/show.html.erb:22]: If we dont use the image tag, its good practice to either remove it or add default values so it doesnt mess with the accessibility of the app :)

File: config/routes.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@
   resources :comments
   resources :photos
   devise_for :users
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  get "/:username" => "users#show", as: :user
```
Comment (JoseMDO) [inline config/routes.rb:11]: nice custom route!

File: app/views/comments/_form.html.erb [hunk 3]
```diff
@@ -1,15 +1,23 @@
...
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
 
-  <div class="form-inputs">
-    <%= f.association :author %>
-    <%= f.association :photo %>
-    <%= f.input :body %>
+  <%= form.hidden_field :photo_id %>
```
Comment (JoseMDO) [inline app/views/comments/_form.html.erb:14]: great job using hidden inputs!

**Review by JoseMDO — APPROVED** [review]
Great job Brandon!

File: app/views/comments/_form.html.erb [hunk 4]
```diff
@@ -1,15 +1,23 @@
...
+
+  <div class="mb-3">
+    <%= form.text_area :body, class: "form-control" %>
   </div>
 
-  <div class="form-actions">
-    <%= f.button :submit %>
+  <div>
+    <%= form.submit class: "btn btn-outline-secondary btn-block" %>
```
Comment (YoungSong99) [inline app/views/comments/_form.html.erb:21]: Great choice for button design!

File: lib/tasks/dev.rake [hunk 5]
```diff
@@ -50,7 +50,7 @@ task sample_data: :environment do
     Photo.create(
       image: "https://picsum.photos/400",
       owner: user,
-      caption: "Oh we IN HERE IN HERE",
+      caption: Faker::Games::StreetFighter.quote,
```
Comment (YoungSong99) [inline lib/tasks/dev.rake:53]: We've lost funny caption

**Review by YoungSong99 — APPROVED** [review]
Thanks for the clean code!

**Review by JoseMDO — APPROVED** [review]

File: lib/tasks/dev.rake (hunk 5 above)
Comment (brvarner) [reply lib/tasks/dev.rake:53]: Next time! 😅

**Pull request merged**